docs/tutorials/uniq-factories/creating-uniq-factories/index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: 'Introduction'
+
+order: 1
+outline: [0, 4]
+---
+
+# Creating Uniq Factories
+
+We will be walking you through the step by step process of creating your own Uniq Factory on Ultra's blockchain using our very own [Uniq NFT Standard](../../../blockchain/contracts/nft-contract/index.md). For the sake of these guides, we'll be using the Ultra Testnet network.
+
+![](/images/token-factories/intro.png)
+
+## Prerequisites
+
+-   A basic understanding of [Uniq Factories and Uniqs](../../uniq-factories/index.md).
```

Comment:
For some reason when this link is used it goes to `***/tutorials/uniq-factories/` and not  `***/tutorials/uniq-factories/index` and the sidebar does not have `Overview` page selected